### DISCUSSION TITLE
Sort

### DISCUSSION THREAD

**Discussion by Salento78** [microsoft-search/pnp-modern-search#3843] [Q&A]
Hi, I'm trying to add sorting on a numeric column, but it seems the sorting isn't working
Can someone help me?
Thank you

**Comment by wobba** [comment]
Step 1, ensure the crawled property matching your number column is mapped to for example RefinableInt00 if it's an integer. Step2, use that property for sorting. Take a look at https://learn.microsoft.com/en-us/microsoft-365/community/how-do-site-columns-become-managed-properties-thus-available-for-search to get started.
